Update readme.md

ronika-das opened the pull request:
Fixes #654 

A couple of doubts:
Q1. Under which heading to put the following?
  a. [Twilio Quest Exercise Outline](/api/twilio-api.md)
  b. [Data Visualization](/data-visualization/data-visualization.md)
  c. [Assessment 7.9: Data Visualization](/data-visualization/data-viz-assessment.md)

Q2  I have added all assessments as well. Is that alright?

alodahl commented:
Great question! Those should all go at the very end of the client-side section.  This looks excellent, thank you so much! Assessments should be in there unless it’s a replit link, yes. Also, isn’t there one more career week link?

manaswinidas commented on README.md:
```diff
@@ -34,11 +34,11 @@ If you would like to contribute to the curriculum, please [sign up here](https:/
 1. [Ergonomics](/ergonomics/ergonomics.md)
+1. [Roles in Tech](/roles-in-tech/roles-in-tech.md)
+1. [Tech Industry Tips](/welcome-tips-rules/tech-industry-tips.md)
+1. [Growth Mindset](/growth-mindset/YCLA_LessonPlan_v10.pdf)
```
All points start from number 1 :sweat_smile: Is it meant to be this way?

ronika-das replied:
in the actual readme.md, the 1 gets inccremented
thats the syntax

manaswinidas commented:
Should we make another assessment session or something for the exercises? As all of the links are exercise and/or assessment links

ronika-das commented:
@alodahl  i will do the necessary changes. I checked all the links,they are alright

ronika-das commented:
the following are not being shown as links:
a. [Restaurant Recommender System with Javascript](/object-oriented-programming/restaurant recommender/RestaurantRecommender - part1.md)
b. [Restaurant Recommender System with Javascript - Part 2](/object-oriented-programming/restaurant recommender/RestaurantRecommender-part2.md

When I hover over the directory, it says path to empty directories

Tech Industry tips and Welcome ,Tips Rules were the same link. i removed the former

alodahl commented:
@ronika-das, the files are there... see if you can fix them. There might be an extra space or something. https://github.com/Techtonica/curriculum/blob/master/object-oriented-programming/restaurant%20recommender/RestaurantRecommender-part2.md

alodahl commented:
@manaswinidas no, everything should be in chronological order- ronika has it right.  Thanks for noticing, though.

alodahl commented:
@ronika-das I found the welcome tips link! 
https://docs.google.com/document/d/1SWiUl0lYQ0sYDM8vQnVhJVOvT4MRA-9pDglKOgqTWak
Can you please put the tech tips link back?

ronika-das commented:
@alodahl  I added the tech tips.
However unable to add the relative path after several attempts. I tried with spaces etc
1. /object-oriented-programming/restaurant recommender/RestaurantRecommender-part2.md
2.  /object-oriented-programming/restaurant recommender/RestaurantRecommender - part1.md

the object -oriented-programming is an empty directory(that is it has no files only folders). i believe that's why it is not working. this links might be able to provide some insight
a. https://github.blog/2014-08-19-folder-jumping/
b. https://stackoverflow.com/questions/115983/how-can-i-add-an-empty-directory-to-a-git-repository

alodahl commented:
@ronika-das that's annoying, sorry about that!  Could you please:
- put the restaurant recommender/ folder in the javascript folder
-  delete object-oriented-programming/
- make sure any old links to /object-oriented-programming/restaurant recommender/ in the project now go to /javascript/restaurant recommender/  ?

Thanks, you're a champion!

ronika-das commented:
on it!

ronika-das commented:
i have deleted the oops folder and added the contents to javascript.
even then it wasn't working. Turns out that links should not have spaces between them. There was a space between Restaurant Recommender. I avoided this by using %20 instead of spaces. thought i'd share it with you :)

And i have not added this link from the sheet https://github.com/Techtonica/curriculum/blob/master/roles-in-tech/technical-writing.md
I think it was an example. i can add if u want

alodahl commented:
@thank you Ronika!  Please do add technical-writing, as the file will be completed soon.
